File: go/get_phone_numbers.go
package main

import (
    "fmt"
    "os"
)

func main() {
    trunkSid := os.Getenv("TRUNK_SID")
    if trunkSid == "" {
        fmt.Println("Error: TRUNK_SID environment variable is required")
        os.Exit(1)
    }

    fmt.Printf("Getting phone numbers for trunk %s...\n", trunkSid)
    result := get(fmt.Sprintf("/trunks/%s/destination-uris", trunkSid))
    fmt.Println("Phone numbers retrieved successfully!")
    _ = result
} 

File: java/GetPhoneNumbers.java
public class GetPhoneNumbers {
    public static void main(String[] args) throws Exception {
        String trunkSid = System.getenv("TRUNK_SID");
        if (trunkSid == null || trunkSid.isEmpty()) {
            System.err.println("Error: TRUNK_SID environment variable is required");
            System.exit(1);
        }

        System.out.printf("Getting phone numbers for trunk %s...%n", trunkSid);
        String result = _Client.get(String.format("/trunks/%s/destination-uris", trunkSid));
        System.out.println("Phone numbers retrieved successfully!");
    }
} 